sga/backend/validators.py

```diff
@@ -6,8 +10,11 @@
 
 
 def validate_file_extension(field_file):
+    """
+    Validate file extension (valid file extensions are defined in VALID_FILE_UPLOAD_EXTENSIONS)
+    """
     ext = os.path.splitext(field_file.name)[1]  # .splitext returns (root, ext)
     if ext.lower() not in VALID_FILE_UPLOAD_EXTENSIONS:
         message = ("Only the following file types are supported: " +
                    "{extensions}".format(extensions=",".join(VALID_FILE_UPLOAD_EXTENSIONS)))
```

Comment:
this can be written better using string literals (that are concatenated at compile time, just as in C):
```
message = ( "Only the following file types are supported: " 
 "{extensions}").format(extensions=",".join(VALID_FILE_UPLOAD_EXTENSIONS))
```